00-HTML-CSS-basics/css/2-6/css/styles.css

```diff
@@ -0,0 +1,85 @@
+a {
+	color: #000000;
+	font-weight: 700;
+	text-decoration: none;
+}
+
+h3 {
+	margin: 5px 0px;
+}
+
+img {
+	max-width: 100%;
+
+	-ms-filter: grayscale(1);
+	-webkit-filter: grayscale(1);
+	-moz-filter: grayscale(1);
+	-o-filter: grayscale(1);
+	filter: grayscale(1);
+
+	-webkit-transition: all 0.3s ease;                  
+    -moz-transition: all 0.3s ease;                 
+    -o-transition: all 0.3s ease;   
+    -ms-transition: all 0.3s ease;          
+    transition: all 0.3s ease;
+}
+
+figure {
+	margin: 0;
+	padding: 0;
+	position: relative;
+}
+
+figcaption {
+	background-color: rgba(255,255,255,0.8);
+	left: 0px;
+	opacity: 0;
+	padding: 10px 0;
+	position: absolute;
+	top: 0px;
+	width: 100%;
+
+	-webkit-transition: all 0.3s ease;                  
+    -moz-transition: all 0.3s ease;                 
+    -o-transition: all 0.3s ease;   
+    -ms-transition: all 0.3s ease;          
+    transition: all 0.3s ease;
+}
+
+figcaption a, figcaption h3 {
+	margin: 0px 20px 5px 20px;
```

Comment:
This should be 0 20px 5px; instead of 0px 20px 5px 20px;, because it's the same definition but more clean.